src/frontend/components/Chat/index.tsx

```diff
@@ -22,23 +22,11 @@ export const Chat = ({ video: baseVideo, standalone }: ChatProps) => {
     }
   }, [video.live_state]);
 
-  const conditionalProps: Partial<BoxExtendedProps> = {};
-  if (standalone) {
-    conditionalProps.height = 'large';
-  } else {
-    conditionalProps.fill = true;
-  }
-
-  return (
-    <Box {...conditionalProps}>
-      {!!standalone ? (
-        <div>
-          Instructor Chat
-          <StudentChat />
-        </div>
-      ) : (
-        <StudentChat />
-      )}
+  return !!standalone ? (
```

Comment:
And what about adding a default value to false for this props ? `export const Chat = ({ video: baseVideo, standalone = false }: ChatProps) => {`

And then you can replace here with `  return standalone ? (`

Reply:
done